### DISCUSSION TITLE
how to build a cross-platform application and put it on the server so that the browser part works

### DISCUSSION THREAD

**Discussion by al-zakaria** [AvaloniaUI/Avalonia#17382] [Q&A]
I have a cross-platform application and I need to put it on the server? The most important thing for me is that it works correctly in the browser

**Comment by maxkatz6** [comment]
This page should help you how to build Browser version of the app: https://docs.avaloniaui.net/docs/guides/platforms/how-to-use-web-assembly

On how to host it on your server, it depends on the server or hosting itself.
For github, for example, it would be https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site
Note, you need to disable jekyll there, as GitHub doesn't do it for you: https://github.blog/news-insights/bypassing-jekyll-on-github-pages/. Otherwise some files will be missing in runtime.
